frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostGroupsModal.cy.tsx

```diff
@@ -30,67 +30,97 @@ describe('Inventory Host Groups List', () => {
     });
 
     it('Inventory Groups Add Modal Renders', () => {
+      cy.intercept(
+        {
+          method: 'OPTIONS',
+          url: '/api/v2/inventories/1/groups/',
+          hostname: 'localhost',
+        },
+        {
+          fixture: 'mock_options.json',
+        }
+      ).as('getFilterOptions');
       cy.mount(<InventoryHostGroupsAddModal {...props} />, {
         path: '/inventories/:inventory_type/:id/hosts/:host_id/*',
         initialEntries: ['/inventories/inventory/1/hosts/1/groups'],
       });
-      cy.get('tbody').find('tr').should('have.length', 10);
+      cy.get('tbody').find('tr').should('have.length', 4);
```

Comment:
The amount of lines comes from the fixture? Might be worth to change it to a const somewhere? WDYT?